Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -6188,6 +6193,43 @@ void LocalFrameView::scrollbarWidthChanged(ScrollbarWidth width)
     m_needsDeferredScrollbarsUpdate = true;
 }
 
+int LocalFrameView::scrollbarGutterWidth(bool isHorizontalWritingMode) const
+{
+    if (verticalScrollbar() && verticalScrollbar()->isOverlayScrollbar())
+        return 0;
+
+    if (!verticalScrollbar() && !(scrollbarGutterStyle().isAuto || ScrollbarTheme::theme().usesOverlayScrollbars()) && isHorizontalWritingMode)
+        return ScrollbarTheme::theme().scrollbarThickness(scrollbarWidthStyle());
+
+    if (!verticalScrollbar())
+        return 0;
+
+    return verticalScrollbar()->width();
+}
+
+
+IntSize LocalFrameView::computeScrollbarGutter() const
```

Comment:
It's a bit unclear from the function name what this returns. Better to call it `spaceForScrollbarGutters()`.

We really need a term for "scrollbar or scrollbar gutter". Maybe it's "scrollbar space". So this would just be `totalScrollbarSpace()` or something.

Reply:
Fixed.